src/main/webapp/script.js

```diff
@@ -98,27 +122,53 @@ function createCanvasForId(id) {
 /**
  * Updates the page with new uploaded image.
  * It calls blurring method, puts new and blurred
- * images on canvases and updates blurRadius input
- * bar for new image.
+ * images on canvases, updates blurRadius input
+ * bar for new image and download button for
+ * every new blurred image.
  * @param {String} imageUrl
  * @param {Array<Rect>} blurAreas
- * @return {Promise<void>} nothing
+ * @param {String} imageType
+ * @param {String} imageFileName Name of an uploaded image file.
  */
-async function processImage(imageUrl, blurAreas) {
+async function processImage(imageUrl, blurAreas, imageType, imageFileName) {
```

Comment:
You often pass the image type and the file name together. Maybe it makes sense to put them together in a class/struct, and pass an object of that class instead? Maybe you could also put imageUrl and imageObj there too.